src/pages/wishlist/Wishlist.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useState } from "react";
+import { Navbar } from "../../components/components-index";
+import "./Wishlist.css";
+import axios from "axios";
+import StarRatings from "react-star-ratings";
+import { Link } from "react-router-dom";
+import { useAppContext } from "../../context/AppContext";
+
+export const Wishlist = () => {
+  const encodedToken = localStorage.getItem("token");
+  axios.defaults.headers.common["authorization"] = encodedToken;
+  const [productWishlist, setProductWishlist] = useState();
+  const { appDispatch } = useAppContext();
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/user/wishlist");
+        setProductWishlist(response.data.wishlist);
```

Comment:
suggestion: instead of life, you can name the function and call in useEffect.

Reply:
Both will work the same and DOM size will be equal as current. Will do that from next time! Thanks!